### PR TITLE
Landing Page Tweaks + Add Footer

### PR DESCRIPTION
**Pull request by DannyS03**
- Tweaks to styles
- Re-adds Footer

Overall design to be reconsidered in the near future

### CODE REVIEW

**Comment by DannyS03** [comment]
proposing we update the landing page tabs - while "new" and "experienced" are good categorizations, I feel they don't accurately depict all the content, and the sub-labels on the cards mention the "difficultly"

**Comment by cerberushades** [comment]
> proposing we update the landing page tabs - while "new" and "experienced" are good categorizations, I feel they don't accurately depict all the content, and the sub-labels on the cards mention the "difficultly"

We might also rethink that "Scale with Polygon". I'm not sure all products by polygon deal with scaling. Polygon ID probably not

File: docusaurus.config.js [hunk 1]
```diff
@@ -44,74 +44,126 @@ module.exports = {
   onBrokenLinks: 'log',
   themeConfig: {
     footer: {
-      /*
+      style: 'dark',
       links: [
+        {
+          title: "Scaling Solutions",
```
Comment (cerberushades) [inline docusaurus.config.js:50]: why not keep it only "Solutions"?

File: src/css/custom.css [hunk 2]
```diff
@@ -840,7 +840,7 @@ html[data-theme='dark'] .header-github-link:before {
 
 .show-card {
   /* Light_Gray_700 */
-  background: #ffffff;
+  background: #D0BAF5;
```
Comment (cerberushades) [inline src/css/custom.css:843]: not sure this color provides a good contrast with black... I'd go with sth lighter or keep the current one...

Comment (cerberushades) [reply src/css/custom.css:843]: the navbar, on the other hand, is ok in terms of contrast, but it is calling too much attention! Also, we'd have to check the polygon logo there...

**Review by cerberushades — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (DannyS03) [reply docusaurus.config.js:50]: Didn't include Polygon ID and it follows the footer list on the main site

File: src/css/custom.css (hunk 2 above)
Comment (DannyS03) [reply src/css/custom.css:843]: > > proposing we update the landing page tabs - while "new" and "experienced" are good categorizations, I feel they don't accurately depict all the content, and the sub-labels on the cards mention the "difficultly"
> 
> We might also rethink that "Scale with Polygon". I'm not sure all products by polygon deal with scaling. Polygon ID probably not

right we mentioned this already with Polygon ID but decided to keep it for now

**Comment by DannyS03** [comment]
> > proposing we update the landing page tabs - while "new" and "experienced" are good categorizations, I feel they don't accurately depict all the content, and the sub-labels on the cards mention the "difficultly"
> 
> We might also rethink that "Scale with Polygon". I'm not sure all products by polygon deal with scaling. Polygon ID probably not

right we mentioned this too with Polygon ID, but decided to keep it for now

File: src/css/custom.css (hunk 2 above)
Comment (DannyS03) [reply src/css/custom.css]: Addressed

File: src/css/custom.css [hunk 3]
```diff
@@ -5,7 +5,7 @@
  */
 
 :root {
-  --ifm-color-primary: #003cb2;
+  --ifm-color-primary: #D0BAF5;
```
Comment (cerberushades) [inline src/css/custom.css:8]: We need to check with @Nazeim if he's ok with that new color

**Comment by DannyS03** [comment]
@Nazeim  updated - now we use the proposed shade only for dark mode, and I adjusted the color for light mode to the lightest shade instead of plain white

**Review by cerberushades — APPROVED** [review]

**Pull request merged**